cinema/tests/test_movie_view_set.py

```diff
@@ -0,0 +1,201 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from rest_framework import status
+from rest_framework.reverse import reverse
+from rest_framework.test import APIClient
+
+from cinema.models import Movie, Genre, Actor
+from cinema.serializers import MovieSerializer, MovieListSerializer, MovieDetailSerializer
+
+MOVIE_URL = reverse("cinema:movie-list")
+
+def detail_url(movie_id):
+    return reverse("cinema:movie-detail", args=[movie_id])
+
+def sample_movie(**params) -> Movie:
```

Comment:
This is a code quality issue. You've defined a helper function `sample_movie` which is great. However, it's not clear what this function does based on its name. Consider renaming it to something more descriptive, like `create_sample_movie`.